astropy/units/tests/test_units.py

```diff
@@ -143,12 +146,6 @@ def test_units_conversion():
     assert_allclose(u.spat.to(u.sr), 12.56637061435917)
 
 
-def test_units_manipulation():
-    # Just do some manipulation and check it's happy
-    (u.kpc * u.yr) ** Fraction(1, 3) / u.Myr
-    (u.AA * u.erg) ** 9
```

Comment:
I didn't see any value in this test. The functionality is checked in other tests.